benchmark.py

```
@@ -0,0 +1,826 @@
+
+
+def percentile(data: list[float], p: float) -> float:
+    if not data:
+        return 0.0
+    # Use statistics.quantiles with method='inclusive' for better compatibility
+    quantiles = statistics.quantiles(data, n=100, method='inclusive')
+    # quantiles[0] is 1st percentile, quantiles[94] is 95th percentile  
+    index = int(p) - 1
+    if index < 0:
+        return min(data)
+    elif index >= len(quantiles):
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)